conf/application.conf

```diff
@@ -0,0 +1,11 @@
+########### Modules to start ###########
```

Comment:
All those files are not in the `src/test/resources` folder :) Please remove them.